test/io/ferguson/assoc-domains.chpl

```diff
@@ -36,7 +36,7 @@ proc testBinary(seeds: domain(int)) {
   writeln("Read: ", po.sorted());
 }
 
-proc test(seeds: domain(int)) {
+proc test(in seeds) {
```

Comment:
Did preserving the formal type here break something?

Reply:
No, it's just simpler syntax and style. The old code used default intent which would not compile, because `seeds` would be `const ref` default intent. `seeds` is modified in the body of `testText` and `textBinary` so that's why an `in` is used, although a `ref` could be used too, although that makes the `test` caller get the side effects.

Reply:
`test` should be a generic test "driver" regardless of the types, which is why the types were removed and only the `in` intent was included.

The `testText` and `testBinary` were kept with their formal types, just having `ref` added.

Reply:
I don't think developers should be changing the style that a test was written in unless it directly relates to their PR's focus, or the test author agrees with the change.  Our focus should be on keeping tests running rather than morphing them into our preferred style along the way.

Reply:
I'm not convinced this comment was resolved so much as ignored...  It's admittedly a small thing, but I do have concerns about changing code in cases where different developers may have different opinions about what constitutes best style (e.g., brevity vs. precision)—particularly when it's orthogonal to the PR in question.